src/styles/global.scss

```diff
@@ -138,6 +140,12 @@ body {
 .catch-phrase{
   font-family: Zilla Slab, san-serif, serif;
   font-style: italic;
-  color: white;
+  color: white
 
 }
+
+.App > div:first-child {
+  padding-bottom: 1.5rem;
```

Comment:
Why are you using `rem`s here?

Reply:
I deleted this...I am though using it in the App.module.scss file to make it more responsive